sass/_components.scss

```diff
@@ -803,4 +803,173 @@
+//         </article>
+//         <article class="test column is-2-5">
+//             <img src="https://lorempixel.com/1600/1600/abstract" />
+//         </article>
+//     </div>
+//     <div class="columns content">
+//         <article class="test2 column is-5">
+//             <h2>Latest Video</h2>
+//             <h1>Looking under the hood gold</h1>
+//             <p>Curabitur et nisi venenatis, mattis tortor id, lacinia nulla. Donec id quam velit. Proin bibendum justo vitae tempor aliquam. Aenean tempus augue a dapibus vestibulum. Nullam lacinia magna eget sapien pretium, sit amet venenatis dolor efficitur. Curabitur tristique tincidunt ligula, quis scelerisque diam dapibus eu. Fusce non accumsan tellus. Nunc bibendum augue vitae nibh laoreet accumsan. Nam vitae placerat nulla, ut tempus ipsum. In mattis nisl id felis lacinia interdum. Sed ac enim quis arcu mattis auctor et in nunc.</p>
+//             <button class="icon-button large hide-text fa-info">More information</button>
+//             <button class="icon-button invert large fa-play">24:45</button>
+//         </article>
+//         <article class="test2 column is-3 is-offset-0-5">
+//             <h2>Latest Video</h2>
+//             <h1>Looking under the hood gold</h1>
+//             <p>Curabitur et nisi venenatis, mattis tortor id, lacinia nulla. Donec id quam velit. Proin bibendum justo vitae tempor aliquam. Aenean tempus augue a dapibus vestibulum. Nullam lacinia magna eget sapien pretium, sit amet venenatis dolor efficitur. Curabitur tristique tincidunt ligula, quis scelerisque diam dapibus eu. Fusce non accumsan tellus. Nunc bibendum augue vitae nibh laoreet accumsan. Nam vitae placerat nulla, ut tempus ipsum. In mattis nisl id felis lacinia interdum. Sed ac enim quis arcu mattis auctor et in nunc.</p>
+//             <button class="icon-button large hide-text fa-info">More information</button>
+//         </article>
+//         <article class="test2 column is-3 is-offset-0-5">
+//             <h2>Latest Video</h2>
+//             <h1>Looking under the hood gold</h1>
+//             <p>Curabitur et nisi venenatis, mattis tortor id, lacinia nulla. Donec id quam velit. Proin bibendum justo vitae tempor aliquam. Aenean tempus augue a dapibus vestibulum. Nullam lacinia magna eget sapien pretium, sit amet venenatis dolor efficitur. Curabitur tristique tincidunt ligula, quis scelerisque diam dapibus eu. Fusce non accumsan tellus. Nunc bibendum augue vitae nibh laoreet accumsan. Nam vitae placerat nulla, ut tempus ipsum. In mattis nisl id felis lacinia interdum. Sed ac enim quis arcu mattis auctor et in nunc.</p>
+//             <button class="icon-button large hide-text fa-info">More information</button>
+//         </article>
+//     </div>
+// </section>
+// 
+// Styleguide 5.8
+.tri-hero {
+    $rotation: 25deg;
+    $scaleFactorNarrowColumn: 1.73;
+    $scaleFactorWideColumn: 1.52;
+    // Scale calculation - cos(θ) + max{(W/H),(H/W)} * sin(θ)
```

Comment:
Gist is the tri hero isimage containers rotated 25 degrees with the images inside rotated back 25 degrees. This gives us "slanted" images. The text is in a separate layer on top of the images.